### PR TITLE
feature/eap-fast-integration2

### PR DESCRIPTION
**Pull request by jrouzierinverse**
## Description

Integrate eap fast 
Allow EAP types to be configured from the GUI
## Impacts

freeradius config
## NEWS file entries

New Features
++++++++++++
- EAP type configuration from the admin GUI
## Delete branch after merge

NO

### CODE REVIEW

File: conf/radiusd/eap.conf.example [hunk 1]
```diff
@@ -822,4 +1206,49 @@ eap {
...
+   	#
+		authority_identity = "1234"
+
+   	#  PAC Opaque encryption key (must be exactly 32 bytes in size)
+   	#
+   	#  You MUST change this to a random key for security.  Try using:
+   	#
+   	#  $ od -vAn -N16 -tx8 < /dev/urandom | sed 's/ //g'
+   	#
+		pac_opaque_key = "0123456789abcdef0123456789ABCDEF"
```
Comment (julsemaan) [inline conf/radiusd/eap.conf.example]: This should be taking the value from pf.conf

File: conf/radiusd/eap.conf.example [hunk 2]
```diff
@@ -822,4 +1206,49 @@ eap {
...
+
+   	#  Same as for TTLS, PEAP, etc.
+   	#
+		virtual_server = inner-tunnel
```
Comment (julsemaan) [inline conf/radiusd/eap.conf.example]: This should use packetfence-tunnel

**Review by julsemaan — CHANGES_REQUESTED** [review]

File: conf/radiusd/eap.conf.example (hunk 2 above)
Comment (jrouzierinverse) [reply conf/radiusd/eap.conf.example]: fixed

File: conf/radiusd/eap.conf.example (hunk 1 above)
Comment (jrouzierinverse) [reply conf/radiusd/eap.conf.example]: fixed

**Review by julsemaan — APPROVED** [review]

**Comment by louismunro** [comment]
Please rebase

**Pull request merged**